# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/seamonkey/uname-wrapper.patch
# Copyright (C) 2014 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

The Python scripts fail with warning output on stderr, ...

	- Rene Rebe <rene@exactcode.de>

--- firefox-64.0/toolkit/crashreporter/google-breakpad/src/build/common.gypi.vanilla	2019-02-25 19:45:27.366949238 +0100
+++ firefox-64.0/toolkit/crashreporter/google-breakpad/src/build/common.gypi	2019-02-25 19:45:36.998949638 +0100
@@ -46,7 +46,7 @@
             # else gets passed through, which probably won't work very well; such
             # hosts should pass an explicit target_arch to gyp.
             'host_arch%':
-              '<!(uname -m | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/")',
+              '<!(uname -m 2>/dev/null | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/")',
           }, {  # OS!="linux"
             'host_arch%': 'ia32',
           }],
--- firefox-64.0/build/gyp_includes/common.gypi.vanilla	2019-02-25 19:47:09.190953460 +0100
+++ firefox-64.0/build/gyp_includes/common.gypi	2019-02-25 19:47:19.606953891 +0100
@@ -117,7 +117,7 @@
             # Anything else gets passed through, which probably won't work very
             # well; such hosts should pass an explicit target_arch to gyp.
             'host_arch%':
-              '<!(uname -m | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/i86pc/ia32/")',
+              '<!(uname -m 2>/dev/null | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/i86pc/ia32/")',
           }],
 
           # Set default value of toolkit_views based on OS.
